components/pricing/Pricing.js

```diff
@@ -0,0 +1,5 @@
+const Pricing = () => {
+  return <div>hello pricing</div>;
```

Comment:
instead of dummy text, add something like: `Pricing page in under development...`